package/yast2-packager.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Jun 15 11:07:50 UTC 2016 - igonzalezsosa@suse.com
+
+- Allow installation via HTTPS using a self-signed certificate
```

Comment:
I'd add `... when "ssl_verify=no" boot option is used`.

Just to be clear that we by default do not ignore certificate issues, someone might get scared about a possible security issue... :wink: